OAK-Marketplace-Emotion-Recognition/main.py

```diff
@@ -0,0 +1,24 @@
+import os.path as osp
+from argsparser import parse_args
+from utils.inference import build_model, process_cam
+from utils.helpers import save_video
+from utils.visualization import EmotionBarAnalyzer
+import cv2
+
+
+def main():
+    args = parse_args()
+    model_path = osp.join(args.model_path, "checkpoint")
+    model = build_model(model_path)
+    emotion_analyzer = EmotionBarAnalyzer()
+    visualization_results = process_cam(model, emotion_analyzer, show=args.vis)
+    if args.save_video:
+        save_video(visualization_results)
+    statistic_result = emotion_analyzer.create_result_emotion_bar(720, save=args.save_statistics)
+    emotion_analyzer.create_statistic_pie_chart(save=args.save_statistics)
```

Comment:
create_statistic_pie_chart should return anything because `create_result_emotion_bar` returns so for better interface structure:
```python
statistic_result = emotion_analyzer.create_result_emotion_bar(720, save=args.save_statistics)
pie_chart_result = emotion_analyzer.create_statistic_pie_chart(save=args.save_statistics)
```python

Reply:
Move 720 to object attributes in init:

self.whatever_size = whatever_size